aspnetcore/data/ef-rp/intro/samples/StageSnapShots/cu-part3-sorting/Models/Student.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ContosoUniversity.Models
+{
+    public class Student
+    {
+        public int ID { get; set; }
```

Comment:
Curious as to why ID has an upper-case D given that it is short for Identity?

Reply:
Both the NOAD and the OED report that ID is an abbreviation for identity, identification.  cc @tdykstra  (the original author)